### DISCUSSION TITLE
Is there a way to control where a users goes after login?

### DISCUSSION THREAD

**Discussion by s0kil** [digitallyinduced/ihp#1002] [Q&A] [answered]
I'm trying to create an Admin application and when I login I keep getting directed back to the Web application welcome page.

**Comment by s0kil** [comment, marked as answer]
In your `Admin.Controller.Sessions` module, you could customize the `SessionsControllerConfig` instance, like this:
```haskell
instance Sessions.SessionsControllerConfig Admin where
    afterLoginRedirectPath = pathTo ProductsAction
```

View more config options here: https://ihp.digitallyinduced.com/api-docs/IHP-AuthSupport-Controller-Sessions.html#t:SessionsControllerConfig

**Reply by CSchank** [reply]
But what about on a per-case basis?

**Reply by mpscholten** [reply]
That's not directly possible at the moment.

As a workaround: You could create a custom `AfterLoginAction` that when called redirects to the actual user-specific action. Then set `afterLoginRedirectPath = pathTo AfterLoginAction`.
